fleetd/fleetd.go

```diff
@@ -35,6 +35,7 @@ import (
 
 const (
 	DefaultConfigFile = "/etc/fleet/fleet.conf"
+	FleetdDescription = "fleetd is a simple orchestration system for scheduling systemd units in a cluster of machines."
```

Comment:
fleetd is the server component of fleet, a simple orchestration system for scheduling systemd units in a cluster.